config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -33,6 +33,7 @@ presubmits:
         - --cluster=
         - --extract=local
         - --flush-mem-after-build=true
+        - --gcp-node-size=e2-medium
```

Comment:
I'm a bit reluctant to changing presubmit immediately with postsubmits. We should split them into two separate PR.

Also @mborsz as current and next-week oncall - FYI

Reply:
Sure, removed presubmit from this PR, I will create another one for presubmits.

Reply:
Presubmit only PR is available in #19602